src/overlays/actors/ovl_Demo_Effect/z_demo_effect.c

```diff
@@ -1855,7 +1855,7 @@ void DemoEffect_DrawGodLgt(Actor* thisx, PlayState* play) {
 void DemoEffect_DrawLightEffect(Actor* thisx, PlayState* play) {
     DemoEffect* this = (DemoEffect*)thisx;
     u8* alpha;
-    Gfx* disp;
+    u32 disp;
```

Comment:
this can stay `Gfx*` if the cast below becomes dumber:
`disp = (Gfx*)(u32)gEffFlash1DL;`